Assets/MRTK/SDK/Features/UX/Scripts/Collections/GridObjectCollection.cs

```diff
@@ -42,27 +42,56 @@ public OrientationType OrientType
             set { orientType = value; }
         }
 
-        [Tooltip("Specify direction in which children are laid out.")]
+        [Tooltip("Specify direction in which children are laid out")]
         [SerializeField]
         private LayoutOrder layout = LayoutOrder.RowThenColumn;
 
         /// <summary>
-        /// Specify direction in which children are laid out.
+        /// Specify direction in which children are laid out
         /// </summary>
         public LayoutOrder Layout
         {
             get { return layout; }
             set { layout = value; }
         }
 
+
         [SerializeField, Tooltip("Where the grid is anchored relative to local origin")]
         private LayoutAnchor anchor = LayoutAnchor.MiddleCenter;
+
+        /// <summary>
+        /// Where the grid is anchored relative to local origin
+        /// </summary>
         public LayoutAnchor Anchor
         {
             get { return anchor; }
             set { anchor = value; }
         }
 
+        [SerializeField, Tooltip("How the columns are aligned in the grid")]
+        private LayoutHorizontalAlignment colAlignment = LayoutHorizontalAlignment.Left;
```

Comment:
Nit on naming, if only because it has come up in other styles and conventions I've seen in the past.

In general, prefer expanding out words if the delta between the fully expanded out thing isn't that much different between the non-expanded case.

In this example, columnAlignment reads easier than colAlignment (marginally so), and the cost of four additional characters relative to the shortening is not significant.

Taken to the extreme this leads to some really long crappy names (in a lot of Java code people complain about TypedIteratorFactoryImplemantationFactory.java) but there's a reasonable middle ground that we can strike in examples like this.

Reply:
Also, considering that this has already been done with a bunch of other collateral it's fine to pass on this, just thing to think about going forward.